instrumentation/opentelemetry-instrumentation-threading/pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "opentelemetry-instrumentation-threading"
+dynamic = ["version"]
+description = "Threading tracing for OpenTelemetry"
```

Comment:
nit this description makes it sound like the plugin creates spans. Can you reword to be more general?

Reply:
Added more description.